src/Modules/Family/__test__/primaryInfoFormComponent.test.js

```diff
@@ -5,6 +5,6 @@ import { noop } from '../../../Testing';
 
 test('should render without error', () => {
   expect(() => {
-    render(<PrimaryInfoFormComponent register={noop} errors={noop} getValues={noop} />);
+    render(<PrimaryInfoFormComponent register={noop} errors={noop} getValues={noop} setValue={noop} watch={noop} />);
```

Comment:
I would prefer more robust testing around all the work you did on the `checkValue` and the `handleChangeDob` but we can go back to it later.